src/style.css

```
@@ -0,0 +1,56 @@
+body {
+  margin: 0;
+}
+
+.header-main {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-class {
+  height: 40px;
+  width: 40px;
+}
+
+.nav-menu {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-link {
+  display: block;
+  position: relative;
+  line-height: 60px;
+  font-family: Roboto;
```

It needs additional alternative styles.
```suggestion
  font-family: Roboto, Arial, sans-serif;
```